src/medium-editor-md.js

```diff
@@ -26,10 +26,12 @@ module.exports = function (options, callback) {
     options.events = options.events || ["input", "change"];
     callback = callback || options.callback || function () {};
 
-    // Wait until the Medium Editor is inited
-    setTimeout(function() {
+    // Called by medium-editor during init
+    this.init = function(mediumEditorInstance) {
```

Comment:
Awesome! I was looking for such a thing! :smile: 

Few suggestions before testing and merging:

 - use here the `self` variable that was already defined (`var self = this;`), or remove it if it's not needed (not used)
 - please consider the [code style I documented here](https://github.com/IonicaBizau/code-style)
   - [`line: 34` Ternary operators](https://github.com/IonicaBizau/code-style#binary-and-ternary-operators)
   - replace the *long* `mediumEditorInstance` variable name with `meInstance`

Also, add yourself a contributor in `contributors` array (`package.json`). Create it, because probably it doesn't exist.